singlestore/datadog_checks/singlestore/values_cleaner.py

```diff
@@ -0,0 +1,63 @@
+# (C) Datadog, Inc. 2021-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from typing import AnyStr, Callable, Optional, Sequence, Union
+
+from datadog_checks.singlestore.queries import AGGREGATORS, MV_GLOBAL_STATUS
+
+
+class SingleStoreCleaningException(Exception):
+    pass
+
+
+UNITS = {
+    'GB': 1e9,
+    'MB': 1e6,
+    'KB': 1e3,
+    'Bytes': 1,
+    'ms': 1e-3,
+    'ns': 1e-9,
+    'units': 1,
```

Comment:
Just curious, do you know why `units` is a unit?

Reply:
The "license" related metrics (how much licenses units you've used, how much you have left...) are using this special naming